tests/phpunit/tests/block-template-utils.php

```diff
@@ -161,6 +161,53 @@ public function test_build_block_template_result_from_file() {
 		$this->assertEmpty( $template_part->modified );
 	}
 
+	/**
+	 * @dataProvider data_build_block_template_result_from_file_injects_theme_attribute
+	 */
+	public function test_build_block_template_result_from_file_injects_theme_attribute( $filename, $expected_content ) {
+		$template = _build_block_template_result_from_file(
+			array(
+				'slug' => 'single',
+				'path' => __DIR__ . '/../data/templates/' . $filename,
+			),
+			'wp_template'
+		);
+		$this->assertSame( $expected_content, $template->content );
+	}
+
+	public function data_build_block_template_result_from_file_injects_theme_attribute() {
+		$theme = 'block-theme';
+		return array(
+			array(
```

Comment:
Can we add keys to the arrays, both for the datasets and the parameter names? This way, it's clear what each is without having to scroll to check the test method's docblock, and if the tests are run with `--testdox`, the the datasets are clearly labelled.

```php
'a template with a template part' => array(
    'filename' => 'template-with-template-part.html',
    'expected_content' => 'I'm on mobile so I'm not typing block markup 😅',
),
```

Reply:
Good point! Added in 0f0c7c0325.